pkgs/desktops/gnome/extensions/extensionOverrides.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  ddcutil,
+  gjs,
+}:
```

Comment:
```suggestion
{ lib
, ddcutil
, gjs
}:
```

Reply:
I think we had this discussion more than once in the past. Until nixpkgs settles on some coding style that is enforced automatically, I will keep it like this.